### PR TITLE
Check for logged in user on editTrack, deleteTrack and createTrack methods #347

### PR DESCRIPTION
**Pull request by bennychan**

### CODE REVIEW

**Comment by liam-wiltshire** [comment]
ok to test

File: src/controllers/EventsController.php [hunk 1]
```diff
@@ -539,7 +539,11 @@ function ($tag) {
      */
     public function createTrack($request, $db)
     {
-        // Should we not check for a login?
+        // Check for login
+        if (! isset($request->user_id)) {
+            throw new Exception("You must be logged in to create track", 401);
```
Comment (Jean85) [inline src/controllers/EventsController.php]: Typo: "to create a track".

**Comment by bennychan** [comment]
Fixed typo

**Comment by liam-wiltshire** [comment]
Thanks @bennychan looks good, will merge this evening.

**Pull request merged**